app/grandchallenge/emails/tasks.py

```diff
@@ -76,20 +93,29 @@ def send_bulk_email(action, email_pk):
         messages = []
         for recipient in paginator.page(page_nr).object_list:
             user = get_user_model().objects.get(pk=recipient.pk)
+            link = reverse(
+                "profile-update", kwargs={"username": user.username}
+            )
+            html_content = render_to_string(
+                "vendor/mailgun_transactional_emails/action.html",
+                {
+                    "title": subject,
+                    "content": safe(html_body),

```

Comment:
`safe` shouldn't be needed here.

Reply:
Oh yes, you're right. This was a relic of my debugging of earlier implementations that didn't work. Removed! ;-)